modules/@apostrophecms/asset/index.js

```diff
@@ -752,8 +752,7 @@ module.exports = {
         if (!self.shouldRefreshOnRestart()) {
           return '';
         }
-        const script = fs.readFileSync(path.join(__dirname, '/lib/refresh-on-restart.js'), 'utf8');
-        return self.apos.template.safe(`<script data-apos-refresh-on-restart="${self.action}/restart-id">\n${script}</script>`);
+        return self.apos.template.safe(`<script data-apos-refresh-on-restart="${self.action}/restart-id" src="${self.action}/refresh-on-restart"></script>`);
```

Comment:
Refactored to use a route to deliver this script so that we don't run afoul of the `unsafe-inline` content security policy rule.